README.md

```diff
@@ -46,7 +46,6 @@ provider "corellium" {
 resource "corellium_v1project" "backtesting" {
   name = "ios_backtesting"
   settings = {
-    version         = 1
```

Comment:
According to the API, the project's settings don't contain a version field anymore, so we removed it from here too.